### DISCUSSION TITLE
Assign a keyboard command to start recording a clip

### DISCUSSION THREAD

**Discussion by mrbelowski** [helgoboss/helgobox#1220] [Q&A]
This seems like a fairly simple use-case and I'm probably missing something straightforward.

I use Playtime2 while recording / arranging guitar & bass loops. This means that clicking the little 'record' button for a slot item is fiddly (as I prefer to play while dancing / staggering around my room) and I'd like to be able to do it some other way - ideally by just mashing the Space bar or some other key. Eventually I'd like to be able to use a separate controller for this (an old joystick, buy a MIDI footswitch, whatever).

But just the keyboard would be a good start.

I've created this mapping in ReaLearn:

![image](https://github.com/user-attachments/assets/069c4e5c-9644-4eaf-8bd9-d98093a7dba7)

it looks like this should map keypad 'ENTER' to the 'Playtime: Slot transport action -> Active slot -> Record/stop' action but it has no effect - it neither starts a selected slot recording, nor does it stop a running slot recording.

What am I missing? Should it be possible to make this work?

**Comment by helgoboss** [comment]
Two things:

1. Did you enable keyboard input in the "Input" menu?
2. Did you activate a slot in Playtime (by clicking on it)? (You are targeting the currently active slot, according to your mapping).

**Comment by mrbelowski** [comment]
I spent a bit more time with this and have got it to work. To be honest I can't quite remember how I did it but it's clearly possible after some poking around. I have 'enter' on the keypad assigned to start / stop recording, no more faffing with the mouse :)
